src/utils/retry-options.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {BackoffSettings} from 'google-gax/build/src/gax';
+import {GoogleError, ServiceError} from 'google-gax';
+
+// (4=DEADLINE_EXCEEDED, 10=ABORTED, 14=UNAVAILABLE)
+export const RETRYABLE_STATUS_CODES = new Set([4, 10, 14]);
```

Comment:
I replaced the numbers with constants directly defined by the grpc library. But note that 13 doesn't belong here because the way it used to work before was [code 13 is only retried if the error is an rst stream error](https://github.com/googleapis/nodejs-bigtable/blob/b2b2fdae61865a36efc8f1f077276053f41d992f/src/table.ts#L932-L942). This PR just preserves that behaviour.

Reply:
I don't like seeing `error.code === 13` being used multiple places in the code though. It's being used as a [magic number](https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad). It would be better to have something like `error.code === INTERNAL_ERROR`. There's probably an enum you can import from somewhere that has the mappings, but if not, you can define the constant yourself

(I kind of feel the same about these other magic numbers, but if they're onl;y being used in one place, you. probably don't need to make constants for each)

Reply:
I replaced all these copy/paste examples with `grpc.status` enum values as requested.